api/PositionOptions.json

```diff
@@ -0,0 +1,540 @@
+              "notes": "Secure context not required"
+            }
+          ],
+          "edge_mobile": [
+            {
+              "version_added": null
+            },
+            {
+              "version_added": true,
+              "notes": "Secure context not required"
+            }
+          ],
+          "firefox": [
+            {
+              "version_added": "55"
+            },
+            {
+              "version_added": "3.5",
+              "version_removed": "55",
+              "notes": "Secure context not required"
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "55"
+            },
+            {
+              "version_added": "4",
+              "version_removed": "55",
+              "notes": "Secure context not required"
+            }
+          ],
+          "ie": {
+            "version_added": "9",
+            "notes": "Secure context not required"
+          },
+          "opera": [
+            {
+              "version_added": true
+            },
+            {
+              "version_added": "16",
+              "version_removed": true,
+              "notes": "Secure context not required"
+            },
+            {
+              "version_added": "10",
+              "version_removed": "15",
+              "notes": "Secure context not required"
+            }
+          ],
+          "opera_android": [
+            {
+              "version_added": true
+            },
+            {
+              "version_added": "16",
+              "version_removed": true,
+              "notes": "Secure context not required"
+            },
+            {
+              "version_added": "10",
+              "version_removed": "15",
+              "notes": "Secure context not required"
+            }
+          ],
+          "safari": [
+            {
+              "version_added": true
+            },
+            {
+              "version_added": "5",
+              "version_removed": true,
+              "notes": "Secure context not required"
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": true
+            },
+            {
+              "version_added": true,
+              "version_removed": true,
+              "notes": "Secure context not required"
+            }
+          ]
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "enableHighAccuracy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PositionOptions/enableHighAccuracy",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "47"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "47",
+                "notes": "Secure context not required"
```

Comment:
So for this and the remaining entries in this support block, I think we don't need any of the secure context notes, since it's covered by the root of the interface. Instead, I'd suggest dropping the "secure" notes and retaining only support statements show the first version the API appeared (and the latest, if it was completely removed at some point). So for this WebView entry, you'd have:

``` 
{
  "version_added": "5"
}
```